public/static/docs/sidebar.json

```diff
@@ -355,6 +355,25 @@
       }
     ]
   },
+  {
+    "slug": "api-reference",
```

Comment:
we have already three different APIs - `dvc.api`, `dvcx` library + summon (should we move summon into `dvcx` cc @dmpetrov @Suor ?), `repo API` (mirrors CLI). We probably should introduce sections right away.

Reply:
I think `dvcx` is a `dvc.api` extension i.e. its the same API. Not sure whats going on with `summon` though, I though it's just part of `dvc.api` as well? What is `repo API`?

Reply:
`Repo` object that reflects anything you can do with CLI inside a repo.

Reply:
I see. Maybe we should extract this part of the review to separate issues? One for `dvcx` and `summon`, and one about documenting the `Repo` object as an even deeper part of the API.

Reply:
kk, let's keep it simple for now

Reply:
OK I'm adding

```markdown
> Please see also [dvcx](https://github.com/iterative/dvcx) for additional API
> extensions.
```

in the API index, and opened #998 regarding `Repo`.